app/grandchallenge/components/management/commands/add_overlay_segments.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+from django.core.management import BaseCommand
+from django.db.transaction import on_commit
+
+from grandchallenge.components.models import (
+    ComponentInterface,
+    ComponentInterfaceValue,
+)
+from grandchallenge.components.tasks import validate_voxel_values
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("json", type=str)
+
+    def handle(self, *args, **options):
+        json_str = options["json"]
+        mapping = json.loads(json_str)
+        for slug in mapping:
```

Comment:
Would be better to have the slug as a required argument and just do this one interface at a time.